src/xunit.performance.api/XunitBenchmark.cs

```diff
@@ -78,11 +89,26 @@ public static (IEnumerable<ProviderInfo> providers, IEnumerable<PerformanceTestM
                         select provider;
 
                     providers = ProviderInfo.Merge(providers);
+
+                    // Inject implicit pmc counters
+                    foreach (var test in testMessageVisitor.Tests)
+                    {
+                        var metrics = new List<PerformanceMetricInfo>(test.Metrics);
+                        foreach (var performanceMetricInfo in PerformanceMetricInfos)
+                            if (performanceMetricInfo.IsValidPmc)
```

Comment:
That's right. This is a list of PMC, and I make sure I only add the ones that are available on the running platform.